<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>771</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>屏幕声音</string>
     </attribute>
     <widget class="QSlider" name="screenLightSlider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>100</y>
        <width>161</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="screenLightSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="savTimeSlider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>屏幕保护</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="savTimeSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>屏幕亮度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>系统时长</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="startScreenSaveCheckBox">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
     <widget class="QSlider" name="cureTimeSlider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>60</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="cureTimeSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>治疗画面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>150</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>工作模式</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="dayModeCheckBox">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>白天</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="nightModeCheckBox">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>黑夜</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="workModeCheckBox">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>自动</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>300</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>音量调节</string>
      </property>
     </widget>
     <widget class="QSlider" name="systemVoiceSlider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>330</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="systemVoiceSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>330</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>系统音量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>370</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>报警音量</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="alarmVoiceSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>370</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="alarmVoiceSlider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>370</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Aharoni</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>开始冻结</string>
      </property>
     </widget>
     <widget class="QSlider" name="coldSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="coldSpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>冻结时长</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Aharoni</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>冻结屏幕</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="useSysColdTimeCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>系统默认冻结屏幕30秒</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="returnDefLight">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>100</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>恢复默认</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>出厂设置</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>P1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="P1_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="P1_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="P2_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>P2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="P2_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="P3_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="P3_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>P3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="P4_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="P4_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>P4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>6</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密码设置区</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitChange">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>消毒参数</string>
     </attribute>
     <widget class="QTableWidget" name="disinfectTableWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>761</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
